includes/Settings.php

```diff
@@ -275,6 +280,17 @@ public function register(): void {
 			]
 		);
 
+		register_setting(
+			self::SETTING_GROUP,
+			self::SETTING_NAME_SHOPPING_PROVIDER,
+			[
+				'description'  => __( 'Shopping provider', 'web-stories' ),
+				'type'         => 'string',
+				'default'      => '',
```

Comment:
```suggestion
				'default'      => 'none',
				'enum'         => [ 'none', 'woocommerce', 'shopify' ],
```

Let's follow the pattern from ad network.